### PR TITLE
Updating Release v1.7.0

### PR DESCRIPTION
**Pull request by sujil02**
**Description of the change:**

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [x] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: website/content/en/docs/upgrading-sdk-version/v1.7.0.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: v1.7.0
+weight: 998993000
+---
+
+## Add the manager config patch to config/default/kustomization.yaml
+
+The scaffolded `--config` flag was not added to either ansible-/helm-operator binary when [config file](https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html) support was originally added, so does not currently work. The `--config` flag supports configuration of both binaries by file; this method of configuration only applies to the underlying [controller manager](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager), not the operator as a whole. To optionally configure the operator's Deployment with a config file, make the following update to config/default/kustomization.yaml:
+```diff # If you want your controller-manager to expose the /metrics # endpoint w/o any authn/z, please comment the following line. - manager_auth_proxy_patch.yaml + +# Mount the controller config file for loading manager configurations +# through a ComponentConfig type +- manager_config_patch.yaml ```
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.7.0.md:9]:
````suggestion
```diff
# If you want your controller-manager to expose the /metrics
# endpoint w/o any authn/z, please comment the following line.
\- manager_auth_proxy_patch.yaml
+
+# Mount the controller config file for loading manager configurations
+# through a ComponentConfig type
+- manager_config_patch.yaml
```
````

File: website/content/en/docs/upgrading-sdk-version/v1.7.0.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Deprecate support for packagemanifests.
+
+The operator-framework is removing support for the packagemanifests format in the near future. Hence, `operator-sdk generate packagemanifests` and `operator-sdk run packagemanifests` commands are deprecated. In order to migrate packagemanifests to bundles, `operator-sdk pkgman-to-bundle` command can be used. Run `operator-sdk pkgman-to-bundle --help` for more details.
+
+_See [#4838](https://github.com/operator-framework/operator-sdk/pull/4838) for more details._
+
+## (go/v3, ansible/v1, helm/v1) Move `leases.coordination.k8s.io` to its own proxy-role rule
+
+Make the following changes to config/rbac/leader_election_role.yaml:
+```diff rules: \- apiGroups:
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.7.0.md:23]: This and the below diff blocks need to be formatted correctly. See the removed changelog fragments for proper formatting

File: website/content/en/docs/upgrading-sdk-version/v1.7.0.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
++  - get 
++  - list 
++  - watch 
++  - create 
++  - update 
++  - patch 
++  - delete 
++- apiGroups: 
++  - coordination.k8s.io 
++  resources:
```
Comment (varshaprasad96) [inline website/content/en/docs/upgrading-sdk-version/v1.7.0.md:43]: my bad, this diffs shouldn't be looking like this either.. as Eric suggested can you look at the changelog entries and format it?

Comment (sujil02) [reply website/content/en/docs/upgrading-sdk-version/v1.7.0.md:43]: okay

File: website/content/en/docs/upgrading-sdk-version/v1.7.0.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  verbs:
+    - get
+```
+
+_See [#4835](https://github.com/operator-framework/operator-sdk/pull/4835) for more details._
+
+## (go/v3) Set the Makefile's `SHELL` variable to the system's `bash` binary
+
+Importing `setup-envtest.sh` needs bash, so your Makefile's `SHELL` variable should be set to `bash` with error options:
+```diff else GOBIN=$(shell go env GOBIN) endif
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.7.0.md:54]:
````suggestion
```diff
else GOBIN=$(shell go env GOBIN)
endif
````

File: website/content/en/docs/upgrading-sdk-version/v1.7.0.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
++# Setting SHELL to bash allows bash commands to be executed by recipes. 
++# This is a requirement for 'setup-envtest.sh' in the test target. 
++# Options are set to exit when a recipe line exits non-zero or a piped command fails. 
++SHELL = /usr/bin/env bash -o pipefail 
++.SHELLFLAGS = -ec 
++ all: build ```
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.7.0.md:60]:
````suggestion
+ all: build
```
````

File: website/content/en/docs/upgrading-sdk-version/v1.7.0.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+## Deprecate support for packagemanifests.
+
+The operator-framework is removing support for the packagemanifests format in the near future. Hence, `operator-sdk generate packagemanifests` and `operator-sdk run packagemanifests` commands are deprecated. In order to migrate packagemanifests to bundles, `operator-sdk pkgman-to-bundle` command can be used. Run `operator-sdk pkgman-to-bundle --help` for more details.
+
+_See [#4838](https://github.com/operator-framework/operator-sdk/pull/4838) for more details._
+
+## (go/v3, ansible/v1, helm/v1) Move `leases.coordination.k8s.io` to its own proxy-role rule
+
+Make the following changes to config/rbac/leader_election_role.yaml:
+```diff rules: \- apiGroups:
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.7.0.md:28]:
````suggestion
```diff
rules: \- apiGroups:
````

File: website/content/en/docs/upgrading-sdk-version/v1.7.0.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  - ""
+-  - coordination.k8s.io
+  resources:
+  - configmaps
++ verbs: 
+  +- get
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.7.0.md:34]: This and the list lines below need to be moved back to the 0th indent so they show up as a diff

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**